de-webapp/src/main/java/org/iplantc/de/server/rpc/GwtRpcController.java

```diff
@@ -67,6 +67,9 @@ public String processCall(String payload) throws SerializationException {
 
             RPCRequest rpcRequest = RPC.decodeRequest(payload,
                                                       this.remoteServiceClass);
+            if (remoteService instanceof DeGwtRemoteLoggingServiceImpl) {
+                ((DeGwtRemoteLoggingServiceImpl)remoteService).setRequest(getThreadLocalRequest());
```

Comment:
This is mostly beyond my knowledge - I followed some examples of other people implementing GWT's stack trace deobfuscator.   But anyway, this method is coming from extending `RemoteServiceServlet`.